src/lib/registration/clients/inst_scc.rb

```diff
@@ -320,6 +322,8 @@ def start_workflow
     def workflow_start
       log.debug "WFM.Args: #{WFM.Args}"
 
+      Yast::Wizard.ClearContents
```

Comment:
Just a question: Why is this needed here if now the BaseSystemRegistration dialog is ensuring to clear the content?

Reply:
Ah, right, this was my initial test, I forgot to remove it, it works without it.